### DISCUSSION TITLE
Animation?

### DISCUSSION THREAD

**Discussion by johnmiles27** [jordanbaird/Ice#257] [General]
How hard would it be to implement some subtle animation of menubar icons sliding out to the left when Ice is triggered as the app Vanilla does? (example of what I'm talking about here: https://www.macstories.net/roundups/managing-your-mac-menu-bar-a-roundup-of-my-favorite-bartender-alternatives/ ). It's a nice little touch IMO, looks very nice. Maybe even a bit of a fade in too while sliding in would look even better

**Comment by johnmiles27** [comment]
Another thought: maybe it would be useful to have an option for the hidden icons to take the place of the visible ones when revealing them? For situations when visible + hidden icons shown at the same time run into the notch? (yes I know the Ice bar is an option; I just prefer everything as minimal as possible)

**Comment by jordanbaird** [comment]
Sorry for the late response on this. The animation does look nice. I'll try to implement it at some point. I'm going to go ahead and convert this into an issue so it can be marked as a feature request. As for hidden items replacing the visible ones, this is being tracked in #240.
